lib/grandpa/message_handler.go

```diff
@@ -460,6 +479,10 @@ func (s *Service) VerifyBlockJustification(hash common.Hash, justification []byt
 		}
 	}
 
+	if len(fj.Commit.Precommits)+equivocatoryVotersCount <= (2 * len(auths) / 3) {
```

Comment:
nit for the noobs out there including myself 😄, maybe set 2 and 3 as named constants inlined just above so we know why this is as such?

e.g.

```suggestion
	const namedTwo = 2
	const namedThree = 3
	if len(fj.Commit.Precommits)+equivocatoryVotersCount <= (namedTwo * len(auths) / namedThree) {
```

Reply:
Done!